common/ast/src/udfs/udf_transformer.rs

```diff
@@ -127,9 +127,12 @@ impl UDFTransformer {
                     op: op.clone(),
                     expr: Box::new(Self::clone_expr_with_replacement(&**expr, replacement_fn)?),
                 }),
-                Expr::Cast { expr, data_type } => Ok(Expr::Cast {
+                Expr::Cast {
+                    expr, data_type, ..
+                } => Ok(Expr::Cast {
```

Comment:
@b41sh  Hi the pg_style can be set as false?

Reply:
We don't used `pg_style`, i think it's OK here.

Reply:
we can set `pg_style` as false, it is used to distinguish the field names of two different casts.
```sql
select cast(a as int8) from test;
select a::int8 from test;
```
I am working on this modification, you can set it as `false` first.